### ISSUE TITLE
Reisekostenumverteilung

### ISSUE THREAD

**Issue by bastrapp** [ultical/ultical#66]
DFV wishes to manage its Reisekostenumverteilung (RKU) with ultical. What is needed:
- [ ] at registration time each team needs to supply its account information to make sure the money can be tranfered to the team
- [ ] after (one day) the tournament a mail needs to be send out to all teams telling them about their share of the RKU and a mail to the DFV-financial officer

**Comment by nuest** [comment]
Do you also want to make the actual calculation within ultical?

**Comment by bastrapp** [comment]
Yes. I wrote a tool some time ago which is used by DFV right now. It will be easy to integrate.

**Comment by nuest** [comment]
Sounds good. 
What are the travelling distances based on?
